src/main/java/io/jenkins/plugins/remotingopentelemetry/engine/RemotingResourceProvider.java

```diff
@@ -0,0 +1,21 @@
+package io.jenkins.plugins.remotingopentelemetry.engine;
+
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.sdk.resources.Resource;
+import io.opentelemetry.semconv.resource.attributes.ResourceAttributes;
+
+/**
+ * Provides {@link Resource}
+ */
+public class RemotingResourceProvider {
+    /**
+     * @return configured {@link Resource}
+     */
+    public static Resource create() {
+        // TODO: more attributes,
+        // TODO: enable to configure them
+        return Resource.create(Attributes.of(
+                ResourceAttributes.SERVICE_NAME, "Jenkins Remoting"
```

Comment:
ℹ️  Note that on the Jenkins Controller, the `service.name` is configurable with a default value of `jenkins`

Would it be "Jenkins Remoting" or more something like "Jenkins Agent"?
Going one step further, I'm wondering if the `service.name`of agents could eventually be `${jenkinsController.serviceName} Agent`. No need to implement this complex thing for the moment. Would be useful once we have multiple Jenkins Controllers reporter observability data to the same observability backend (Jaeger, Zipkin, Elastic...)



https://github.com/jenkinsci/opentelemetry-plugin/blob/0.14-beta/src/main/java/io/jenkins/plugins/opentelemetry/JenkinsOpenTelemetryPluginConfiguration.java

Reply:
I think "Jenkins Agent" is better for users to understand.
> once we have multiple Jenkins Controllers reporter observability data to the same observability backend.
I didn't take it into consideration, I'll do this in future work, thank you!
ref #42